types/office-js-preview/index.d.ts

```diff
@@ -1077,6 +1077,17 @@ declare namespace Office {
              * For example, an add-in that handles the `ItemSend` event can set `allowEvent` to `false` to cancel sending of the message.
              */
             allowEvent: boolean;
+
+            /**
+             * A string value. When the completed method is used to signal completion of an event handler and if the `allowEvent` option is set to `false`,
```

Comment:
The first sentence should describe the API. Also, in this case the datatype is clear from the signature, so it does not need to be mentioned in the description.

```suggestion
             * When the completed method is used to signal completion of an event handler and if the `allowEvent` option is set to `false`,
```

Reply:
Thanks for the suggestion, Ricky. I'll address this in a subsequent PR.